documentation/translations/coordinating.rst

```diff
@@ -96,12 +231,26 @@ More related tools and projects can be found in the
 
 __ https://github.com/python-docs-translations
 
-How is a coordinator elected?
------------------------------
+
+How should I test my translation?
+---------------------------------
+
+The `dashboard <https://python-docs-translations.github.io/dashboard/metadata.html>`_
+tests translations and uploads error logs.
```

Comment:
```suggestion
provides build warnings and lint errors for translation projects.
```

I'd move this sentence at the end of the section, as it doesn't really answer the question.